src/js/player.js

```diff
@@ -839,6 +839,9 @@ class Player extends Component {
     }
 
     this.techCall_('setCrossOrigin', value);
+    if (this.posterImage) {
+      this.posterImage.$('img').crossOrigin = value;
+    }
```

Comment:
thought: maybe PosterImage should have a crossOrigin getter/setter so that this piece can be moved into it?
```
if (this.posterImage) {
  this.posterImage.crossOrigin(value);
}
```
If we do, we'll probably want to update the createEl below to use this option.